app/controllers/variable_drafts_controller.rb

```diff
@@ -13,6 +14,45 @@ def new
     set_current_form
   end
 
+  def show
+    super
+
+    validate_metadata
+  end
+
+  def publish
+    draft = get_resource.draft
+
+    ingested = cmr_client.ingest_variable(draft.to_json, get_resource.provider_id, get_resource.native_id, token)
+
+    if ingested.success?
+      # get information for publication email notification before draft is deleted
+      Rails.logger.info("Audit Log: Draft #{get_resource.entry_title} was published by #{current_user.urs_uid} in provider: #{current_user.provider_id}")
+      user_info = get_user_info
+      short_name = get_resource.draft['ShortName']
```

Comment:
Variable doesn't have 'ShortName', might make more sense to use `get_resource.short_name` which is on the model and both collections and variables write to.